pkg/plugin/v2/scaffolds/internal/templates/controller/controller_suitetest.go

```diff
@@ -149,6 +149,7 @@ var _ = BeforeSuite(func(done Done) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths: []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
+		ErrorIfCRDPathMissing: true,
```

Comment:
Hi @jamielennox Thanks for your contribution. :+1: 
Couple of changes:
1. This change should only happen for v3. We have decided to include only bug fixes for v2 going forward. Could you please revert the change done in v2.
2. You need to run `make generate` command locally, which generates the testdata (based on your change) and push those changes as well.

Reply:
Done, thanks.